; arrayError.L1
; calls array-error
; The call to print should't work

(:main
  (:main
    0 0
    (rdi <- 5)
    (rsi <- 7)
    (call allocate 2)
    (rdi <- rax)
    (rsi <- 7)
    (call array-error 2)
    (rdi <- rax)
    (call print 1)
    (return)))
